enos/modules/vault_unseal_nodes/scripts/unseal-node.sh

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env bash
+
+binpath=${VAULT_INSTALL_DIR}/vault
+
+IFS="," read -a keys <<< ${UNSEAL_KEYS}
+
+function fail() {
+	echo "$1" 1>&2
+	exit 1
+}
+count=0
+retries=5
```

Comment:
Shouldn't we only have to retry once here? My understanding was the first unseal attempt fails, so shamir unseals take N+1 tries (where N is the number of seals)

If all of that is correct, is there a way to use the existing unseal module twice and expect an error on the first attempt, or do we need this script?

If not, maybe we can simplify this script to do ^.

Reply:
I am sure it exits in the second attempt to unseal, `5` here is just an upper limit after which it would fail with error unsealing